src/main/target/common_post.h

```diff
@@ -405,11 +405,6 @@ extern uint8_t __config_end;
 #define USE_CUSTOM_DEFAULTS_ADDRESS
 #endif
 
-#if !defined(USE_EXTI)
-#undef USE_RX_SPI
```

Comment:
Is this now defined for all targets?

Reply:
This just removes defined relevant for when `USE_EXTI` is NOT defined, which is now never the case.